clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -305,7 +305,7 @@ func (w *waitForControlledPodsRunningMeasurement) handleObjectLocked(oldObj, new
 	if err != nil {
 		return fmt.Errorf("meta key creation error: %v", err)
 	}
-	checker, err := w.waitForRuntimeObject(handledObj, isObjDeleted)
+	c, err := w.waitForRuntimeObject(handledObj, isObjDeleted)
```

Comment:
please revert the name to checker - it's more intuitive

Reply:
Done.